modules/selenium/src/main/java/org/testcontainers/containers/RecordingFileFactory.java

```diff
@@ -0,0 +1,9 @@
+package org.testcontainers.containers;
+
+import org.junit.runner.Description;
+
+import java.io.File;
+
+public interface RecordingFileFactory {
```

Comment:
Would it be OK to change this to `RecordingFileNameFactory` (and also rename the implementation)?

The reason I ask is that changing the filename seems to basically be the intent. The parent folder is already controlled separately, so all the user is really able to do is change the filename (unless I'm being very unimaginative 😄 )

Reply:
Certainly the original intent was to enable the file name to match the test name so that it was more easily identified.

Providing a custom factory, however, (if you want to alter the default we've not provided) means that setting the parent directory elsewhere is not necessary. i.e., you can ignore the vncRecordingDirectory argument.

I'm happy to rename it if you prefer?

Reply:
I wonder if the vncRecordingDirectory should be a property of the factory instead of the browser web driver container?

Reply:
I've added docs now too.

Reply:
Let me know about the above thoughts. That, I think, is the last bit left.

Reply:
^^ I get it, OK. Let's keep as-is. I agree with the idea of setting the parent directory here, but as you say it would be a breaking change so let's save it for another day :)

Thank you also for the docs updates!